tests/api_workflow/mocked_api_workflow_client.py

```diff
@@ -165,7 +166,7 @@ def get_tags_by_dataset_id(self, dataset_id, **kwargs):
 
     def perform_tag_arithmetics(self, body: TagArithmeticsRequest, dataset_id, **kwargs):
         _check_dataset_id(dataset_id)
-        if body.new_tag_name is None or body.new_tag_name is '':
+        if (body.new_tag_name is None) or (body.new_tag_name == ''):
```

Comment:
is this your linter kicking in or does this change, change anything?

Reply:
Pytest shows a warning if you compare strings with `is`. The issue is that `is` tests for reference equality and should therefore not used with strings.